packages/mui/src/stories/core/theme/themes.ts

```diff
@@ -90,6 +90,34 @@ export function getThemeOptions(
           },
         },
       },
+      MuiToggleButton: {
+        styleOverrides: {
```

Comment:
why changing theme ?
this will affect toggleButttonInput component

Reply:
That's ok because ToggleButton component is not used in ToggleButtonInput!